frontend/src/pages/primary/Scope.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+    Box,
+    Button,
+    Checkbox,
+    ClickAwayListener,
+    FormControl,
+    FormControlLabel,
+    FormGroup,
+    Popper
+} from "@mui/material";
+import {ChangeEvent, useState, MouseEvent} from "react";
+
+interface ScopeProps {
+    friends: string[],
+    scope: string[],
+    setScope: (scope: string[]) => void
+}
+
+export default function Scope(props: ScopeProps) {
+    const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
```

Comment:
is that sort of like logical or, either null or HTMLElement ?

Reply:
I dont really like this either but its from the MUI docs example and i dont want to mess with them ;)